call_apply_bind_exercise/callApplyBind.js

```diff
@@ -0,0 +1,73 @@
+function IntroducePerson(fullName) {
+	this.fullName = fullName;
+	this.sayHi = function() {
+		return `This person's name is ${this.fullName}`
+	}
+}
+
+var obj = new  IntroducePerson('Harry Potter')
+
+
+// Answer: arguments and sets
```

Comment:
not exactly - sets are not array-like-objects, they are just a type of object. HTML collections (what you get from $() and document.getElementsByTagName or querySelectorAll are array like objects)